src/main/java/com/example/user_management/api/model/request/RegisterUserRequestModel.java

```diff
@@ -11,7 +11,7 @@
 @Getter
 @Setter
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class CreateUserRequestModel {
+public class RegisterUserRequestModel {
```

Comment:
maybe naming is not the best, it indicating action and looks like more a function name rather than class name.